### PR TITLE
React-Redux Capstone : Crypto World

### PR DESCRIPTION
**Pull request by PrantoshB**
# Hello 👋

hello

---

## In this Project, I have implemented the following:

- [x]  **Interactions**

- Home page
  - When the page loads, the web app shows the list of categories that could be filtered out by some parameter, for example by the category name.
  - Along with the category name, it will display some numeric values per category that come from the API.
  - When the user clicks (or taps) on a category item, the application navigates to the details page.

- Details page
  - In the details page, the web app retrieves data from the API to show detailed data in the category.
  - When the user clicks on the "Back" button (<), the user navigates to the home page.

- [x] Create Test files for functions
- [x] Deploy the project on Render
- [x] Fix Linters Errors
- [x] Follow Git and GitHub flows

### Screenshot of the App:

Screenshot

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @PrantoshB, can you please add the key features section to you readme 👍🏿

File: README.md [hunk 1]
```diff
@@ -1 +1,190 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ]  Customize the App with Images.
+- [ ]  Add Sliders.
+- [ ]  Customize the Desktop Version.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (prolajumokeoni) [inline README.md:193]: -  Your readme is missing an acknowledgments section,  from the project requirements; 

> The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.

- [ ]  Kindly give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/)  in your acknowledgement section👍🏿

File: src/pages/Home.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  const coins = useSelector((state) => state.coins.coins);
+
+  React.useEffect(() => {
+    dispatch(fetchCoinsAction());
+  }, [dispatch]);
+
+  return (
+    <>
+      <Global />
+      <div className="cointop"><p>Stats by cryptocurrencies</p></div>
```
Comment (prolajumokeoni) [inline src/pages/Home.js:20]: - From the requirements: 

> When the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for example by the category name.

- [ ]  Your home page should have a list of categories that could be filtered using given parameters, e.g. crypto name, you can create a search bar to filter cryptos by name 😉

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @PrantoshB 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️

- [ ] Based on the project requirements,  Kindly document your project by following all the project documentation [guidelines](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#project-documentation)

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @PrantoshB 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- Your linters are passing all the checks.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by PrantoshB** [comment]
Thanks, @prolajumokeoni & @emmanuellekamwa for Reviewing my PR.
As it's now approved, I am going to merge it.
Have a Beautiful Day 💐

**Pull request merged**